tox.ini

```diff
@@ -60,9 +61,9 @@ commands =
 
 [testenv:integration]
 description = Run integration tests
-changedir = {toxinidir}/tests/integration/
```

Comment:
why do you remove this, and the following line? Is there any reason blocking the zaza test?

Those two lines are added to avoid file renames like you did in the up above (i.e making a unnecessary nested `tests/integration/tests` directory), and you can simply use `tests/integration`.

Reply:
`tests/integration/tests/` nested directory is needed unfortunately. See https://github.com/openstack-charmers/zaza/blob/b22c2eed4c322f1dfc14ffb2d31e0dd18c911a40/zaza/charm_lifecycle/deploy.py#L38

Reply:
Nice find!